### PR TITLE
marco/feature-NFTRWP-191

### PR DESCRIPTION
**Pull request by marco-calderon-primelab**
- Reset CSS property to avoid resizing and jumping when GIF is present on the Carousel.

Evidence attached:

Gif

### CODE REVIEW

File: src/Components/Dashboard/SendNFT/sendNft.module.css [hunk 1]
```diff
@@ -423,6 +423,7 @@
   border-top-right-radius: 12px;
   border-top-left-radius: 12px;
   pointer-events: none;
+  vertical-align: unset;
```
Comment (biswanathsahoo) [inline src/Components/Dashboard/SendNFT/sendNft.module.css]: @marco-calderon-primelab We're using scss in this project so if there is any changes in css then do it in .scss file and .css file will be generated automatically from the .scss file.

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

File: src/Components/Dashboard/SendNFT/sendNft.module.css (hunk 1 above)
Comment (marco-calderon-primelab) [reply src/Components/Dashboard/SendNFT/sendNft.module.css]: Sure, I'll make the proper adjustment.

**Review by marco-calderon-primelab — COMMENTED** [review]
Removed property from CSS file, added it into SCSS

**Review by biswanathsahoo — APPROVED** [review]

**Comment by biswanathsahoo** [comment]
@marco-calderon-primelab there is a conflict. Please resolve this

**Review by biswanathsahoo — APPROVED** [review]

**Pull request merged**